tools/cdk-build-tools/bin/cdk-build.ts

```diff
@@ -54,12 +54,12 @@ async function main() {
 const timers = new Timers();
 const buildTimer = timers.start('Total time');
 
-main().then(() => {
-  buildTimer.end();
-}).catch(e => {
-  buildTimer.end();
+main().catch(e => {
   process.stderr.write(`${e.toString()}\n`);
-  process.stderr.write(`Build failed. ${timers.display()}\n`);
+  process.stderr.write('Build failed.');
   process.stderr.write('!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n');
   process.exit(1);
+}).finally(() => {
+  buildTimer.end();
+  process.stdout.write(`Build times for ${currentPackageJson().name}: ${timers.display()}\n`);
```

Comment:
Would it make more sense to accumulate (eg in a file) and print a summary report at the end of the full build?

Otherwise I can't really see how this is usable

Reply:
> Otherwise I can't really see how this is usable

Download the build log and run a grep on 'Build times ' ?